package.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Should we go ahead and add "engines", defining which versions of Node we plan to support?

Reply:
We decided to punt on this for now, choosing to reexamine this before the release. Our main concern is that we don't want to be too heavy-handed in enforcing particular versions of Node, despite us only officially supporting LTS releases and @latest.